core/app/mailers/spree/order_mailer.rb

```diff
@@ -7,7 +7,7 @@ def confirm_email(order, resend = false)
       @store = @order.store
       subject = build_subject(t('.subject'), resend)
 
-      mail(to: @order.email, from: from_address(@store), subject: subject)
+      mail(to: @order.email, bcc: bcc_address(@store), from: from_address(@store), subject: subject)
```

Comment:
When there is no `bcc_address` present, would this cause any issues?

Reply:
ActionMailer only uses the bcc if it's not `blank?` so nil or empty strings don't break anything
![image](https://user-images.githubusercontent.com/5720486/83352312-209a2a80-a310-11ea-9c79-a8e749309d49.png)

Reply:
perfect!